impl/test/src/test/resources/workflows-samples/container/container-cleanup-default.yaml

```diff
@@ -0,0 +1,12 @@
+document:
+  dsl: '1.0.2'
+  namespace: test
+  name: container-cleanup-default
+  version: '0.1.0'
+do:
+  - runContainer:
+      run:
+        container:
+          image: alpine:latest
```

Comment:
We can use `busybox` instead: https://hub.docker.com/_/busybox

It's less than 1MB.